pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -191,7 +191,7 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 			}
 		}
 
-		ibmcloudConfig, err := ibmcloudmanifests.CloudProviderConfig(clusterID.InfraID, accountID, installConfig.Config.IBMCloud.Region, controlPlane.Zones, compute.Zones)
+		ibmcloudConfig, err := ibmcloudmanifests.CloudProviderConfig(clusterID.InfraID, accountID, installConfig.Config.IBMCloud.Region, installConfig.Config.Platform.IBMCloud.ClusterResourceGroupName(clusterID.InfraID), controlPlane.Zones, compute.Zones)
```

Comment:
This line is getting complex. Please set the the resourceGroupName to a variable and use it here. ie:
```
resouceGroupName := installConfig.Config.Platform.IBMCloud.ClusterResourceGroupName(clusterID.InfraID)
ibmcloudConfig, err := ibmcloudmanifests.CloudProviderConfig(clusterID.InfraID, accountID, installConfig.Config.IBMCloud.Region, resourceGroupName, controlPlane.Zones, compute.Zones)
```